### DISCUSSION TITLE
Public access S3 upload

### DISCUSSION THREAD

**Discussion by jasonford1** [Budibase/budibase#11900] [Help]
Hi team,

Big fan of your work, but I have hit a wall and I need to know if my use-case is in scope for Budibase.

Context: Customers need to upload photos of their bikes so I can give them service quotes.

S3 Upload component says "Session not authenticated" when a Public user accesses the form.

Attachment component allows for the upload, but then I cannot share back the photos they uploaded because public access to attachments yields "access denied."

_**Can the public do S3 Upload? Can the public access budibase attachments?**_

Regards,
Jason

Budibase Cloud
TenantID: newwheel

**Comment by jasonford1** [comment]
Submitted an issue since this is either a bug or unsupported feature and not a discussion topic.

**Comment by torbenraab** [comment]
Was there any update on this? Same issue here

**Comment by jasonford1** [comment]
The dev team told me it was an unsupported use case and I moved my project
to Appsmith.
